src/components/organisms/UiMenu/UiMenu.vue

```diff
@@ -62,139 +53,179 @@ export interface MenuProps {
    */
   items?: (string | MenuRenderItem)[];
   /**
-   * Use this props to allow using arrows to navigation.
+   * Use this props to allow using arrows instead tabs for navigation UiMenuItems.
    */
   enableKeyboardNavigation?: boolean;
+  /**
+   * Use this props to pass refs to UiMenuItems.
+   */
+  itemsTemplateRefs?: ComponentInstance<typeof UiMenuItem>[] | [];
 }
 export type MenuAttrsProps = DefineAttrsProps<MenuProps, ListAttrsProps>;
-export interface MenuItem {
-  $el: HTMLLIElement,
-  tabindex: Ref<number | null>
-}
 
 const props = withDefaults(defineProps<MenuProps>(), {
   items: () => ([]),
-  enableKeyboardNavigation: false,
+  enableKeyboardNavigation: true,
+  itemsTemplateRefs: () => ([]),
 });
-const menuItemAttrs = ({
-  /* eslint-disable-next-line @typescript-eslint/no-unused-vars */
-  name, label, ...itemAttrs
-}: MenuRenderItem) => itemAttrs;
-const itemsToRender = computed<MenuRenderItem[]>(() => (props.items.map((item, key) => {
+provide('enableKeyboardNavigation', props.enableKeyboardNavigation);
+const itemsToRender = computed<MenuRenderItem[]>(() => (props.items.map((item, index) => {
   if (typeof item === 'string') {
     return {
-      name: `menu-item-${key}`,
+      name: `menu-item-${index}`,
       label: item,
     };
   }
   return {
-    name: `menu-item-${key}`,
+    name: `menu-item-${index}`,
     ...item,
   };
 })));
-const menuItems = ref<MenuItem[] | []>([]);
-provide('menuItems', menuItems);
-const mItems = computed(() => (menuItems.value));
+const disabledKeyboardNavigation = computed(() => (!props.enableKeyboardNavigation));
```

Comment:
This seems very unnecessary to me.

Reply:
It's used in 3 places and looks better than `!props.enableKeyboardNavigation` in conditions. Why it's unnecessary?

Reply:
Does it? In code below where you use
```
if (disabledKeyboardNavigation)
if (props.enableKeyboardNavigation)
```

It implies it's two different things. Personally I don't see a reason to use both `enabled` and `disabled` at the same time.

Reply:
ok 👍  is removed